aten/src/ATen/cpu/vec/vec256/zarch/vec256_zarch.h

```diff
@@ -1260,10 +1260,10 @@ DEFINE_CLAMP_MAXMIN_FUNCS(double)
 #if !defined(vec_float) || __ARCH__ < 13
 #warning \
     "float->int and int->float conversion is simulated. compile for z15 for improved performance"
-ZSimdVect<float> vec_int_flt(const ZSimdVect<int> x) {
+inline ZSimdVect<float> vec_int_flt(const ZSimdVect<int> x) {
```

Comment:
This feels wrong, as veclib is header only library implemented in anonymous namespace